src/quick-lint-js/lex.h

```diff
@@ -108,6 +108,11 @@ class lexer {
   // Postcondition: this->peek().type == token_type::regexp.
   void reparse_as_regexp();
 
+  // Returns true if a valid regexp literal comes after '*/' or '**/'
+  // Precondition: this->input[offset+1] == '/'
+  // offset is number of characters preceding the final /
```

Comment:
"the final /" is confusing phrasing. It could mean the end of the regexp literal (which has two `/` characters: the initial `/` and the final `/`).
```suggestion
  // offset is number of characters preceding the /
```